### PR TITLE
App

### PR DESCRIPTION
**Pull request by anxolin**
Fixing some TODOS left in the code :) 

This one adds the APP_ID. It's loaded from env, defaulted to 0. Also included as a header in all requests to the operator

Also, added in travis 
image

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr62--gpv2swap.review.gnosisdev.com)**: Testing web app

File: src/custom/constants/index.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { ChainId } from '@uniswap/sdk'
 
-// TODO: Load app Id
-export const APP_ID = 0
+export const APP_ID = Number(process.env.REACT_APP_ID || 0)
```
Comment (W3stside) [inline src/custom/constants/index.ts:3]: dont need 0 in number constructor

**Review by W3stside — APPROVED** [review]

File: src/custom/utils/operator.ts [hunk 2]
```diff
@@ -13,8 +14,8 @@ const API_BASE_URL: Partial<Record<ChainId, string>> = {
 }
 
 const DEFAULT_HEADERS = {
-  'Content-Type': 'application/json'
-  // TODO: Maybe add a custom header for the AppId (same as the signing tx)
+  'Content-Type': 'application/json',
+  'X-AppId': APP_ID.toString()
```
Comment (Velenir) [inline src/custom/utils/operator.ts:18]: Server expects this?

**Review by Velenir — APPROVED** [review]

File: src/custom/constants/index.ts (hunk 1 above)
Comment (anxolin) [reply src/custom/constants/index.ts:3]: Good for me, I will change. I thought it was more explicit

File: src/custom/utils/operator.ts (hunk 2 above)
Comment (anxolin) [reply src/custom/utils/operator.ts:18]: now it does :)

Comment (anxolin) [reply src/custom/utils/operator.ts:18]: https://github.com/gnosis/gp-v2-services/pull/135

**Comment by anxolin** [comment]
@e00E @fleupold  this PR is waiting for the service to be up with the CORS change. 
Could you confirm when is up sow e can review if this fixes this PR?

image

If you want to test yourself you have this URL https://pr62--gpv2swap.review.gnosisdev.com/

**Comment by fleupold** [comment]
It's working now. The problem was missing CORS config on the ingress config (https://gitlab.gnosisdev.com/devops/gnosis-staging/merge_requests/387). This x-appid is something weird apparently.

**Pull request merged**